client/data/sites/site-excerpt-types.ts

```diff
@@ -0,0 +1,33 @@
+import type { SiteData, SiteDataOptions } from 'calypso/state/ui/selectors/site-data';
+
+// Performance-optimized request for lists of sites.
+// Don't add more fields because you will make the request slower.
+export const SITE_EXCERPT_REQUEST_FIELDS = [
+	'ID',
+	'URL',
+	'is_coming_soon',
+	'is_private',
+	'launch_status',
+	'icon',
+	'name',
+	'options',
+	'plan',
+] as const;
+
+export const SITE_EXCERPT_COMPUTED_FIELDS = [ 'slug' ] as const;
+
+export const SITE_EXCERPT_REQUEST_OPTIONS = [ 'is_wpforteams_site' ] as const;
```

Comment:
These are not really types, so I'd move this to a new file.

Reply:
Just for context, I split these types into a separate file to make them easier to import into the `@automattic/components` package.

A `constants.ts` file for the concrete values makes sense. The interesting thing is that these values are used as part of the type definition for `SiteExcerptData`, so if you import `site-excerpt-types.ts` you end up transitively depending on `constants.ts` anyway. But that works fine.

Reply:
Follow up https://github.com/Automattic/wp-calypso/pull/65877